concordia/settings_dev.py

```diff
@@ -21,10 +21,30 @@
 
 CONCORDIA = {"netloc": "http://0.0.0.0:8000"}
 
-IMPORTER = {"IMAGES_FOLDER": "/tmp/concordia_images/"}
-
 S3_BUCKET_NAME = "concordia-staticpages"
 
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+AWS_STORAGE_BUCKET_NAME = S3_BUCKET_NAME
+AWS_DEFAULT_ACL = None  # Don't set an ACL on the files, inherit the bucket ACLs
+
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
```

Comment:
Do we need to manage these in the settings file versus just letting boto3 pick them up by default?

Reply:
boto3 picks them up from the IAM role in ECS, but when I'm testing locally they're in my environment.

Reply:
Doesn't boto also pick up the environmental variables without us having to put them in the settings file?